docs/src/pages/getting-started/installation/dependencies.flutter.mdx

```diff
@@ -0,0 +1,35 @@
+import { Tabs, TabItem } from '@aws-amplify/ui-react';
+
+Add the `amplify_authenticator` package with [pub](https://pub.dev/):
+
+```shell
+flutter pub add amplify_authenticator
+```
+
+This will add a dependencies like this to your package's pubspec.yaml
+
+```yaml
+dependencies:
+  amplify_authenticator: ^0.1.0-rc.5
```

Comment:
We discussed this with the flutter team. I cannot remember if we decided on 0.1.0 or 1.0.0, but it will not be rc. I can update this since I don't think we will publish the docs until GA.

For context, the amplify flutter libraries currently use 0 based versioning (latest stable is 0.3.2, next major is 0.4.0).